packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -150,6 +141,13 @@ const fetchRelatedTerariumArtifacts = async () => {
 	}
 };
 
+function updateParamaterRow(attribute: string, newParameterRow) {
+	if (model.value && model.value[attribute]) {
```

Comment:
If you want to avoid the double test you can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#syntax) on array too.

```suggestion
	if (model?.value?.[attribute]) {
```